apps/portal/app/components/create-claim/create-claim-popovers.tsx

```diff
@@ -111,30 +100,17 @@ export const IdentityPopover: React.FC<IdentityPopoverProps> = ({
                 <div className="w-80 max-md:w-[80%]">
                   <ProfileCard
                     variant={
-                      selectedIdentity.is_user === true
+                      selectedIdentity?.type ===
+                      ('Account' || 'Person' || 'Default')
                         ? Identity.user
                         : Identity.nonUser
                     }
-                    avatarSrc={getAtomImage(selectedIdentity)}
-                    name={getAtomLabel(selectedIdentity)}
-                    id={
-                      selectedIdentity.is_user === true
-                        ? selectedIdentity.user?.ens_name ??
-                          sliceString(selectedIdentity.user?.wallet, 6, 4)
-                        : selectedIdentity.identity_id
-                    }
-                    stats={
-                      selectedIdentity.is_user === true
-                        ? {
-                            numberOfFollowers:
-                              selectedIdentity.follower_count ?? 0,
-                            numberOfFollowing:
-                              selectedIdentity.followed_count ?? 0,
-                          }
-                        : undefined
-                    }
-                    bio={getAtomDescription(selectedIdentity)}
-                    ipfsLink={getAtomIpfsLink(selectedIdentity)}
+                    avatarSrc={getAtomImageGQL(selectedIdentity)}
+                    name={getAtomLabelGQL(selectedIdentity)}
+                    id={getAtomIdGQL(selectedIdentity)}
+                    stats={undefined} // TODO: add stats when available
```

Comment:
This is available on another query. I split the queries out for revalidation and optimization purposes. It can be obtained via `useGetConnectionsCountQuery` and passed in. 

It's an aggregate query so it's pretty quick. We can update that in a future PR though